reference/composedshader.md

```diff
@@ -0,0 +1,105 @@
+## ComposedShader
+
+```
+ComposedShader {
+  vrmlField MFNode fields []
+  vrmlField MFNode parts  []
```

Comment:
I would keep it to fields as I believe we should stick to the X3D standard as much as possible.